core/files/assets/files-upload-handler.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Elementor\Core\Files\Assets;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Files_Upload_Handler {
+	public function __construct() {
+
+	}
+
+	/**
+	 * is_elementor_media_upload
+	 * @return bool
+	 */
+	protected function is_elementor_media_upload() {
+		return isset( $_POST['uploadTypeCaller'] ) && 'elementor-editor-upload' === $_POST['uploadTypeCaller']; // phpcs:ignore
+	}
+
+	/**
+	 * file_sanitizer_can_run
+	 * @return bool
+	 */
+	public static function file_sanitizer_can_run() {
+		return class_exists( 'DOMDocument' ) && class_exists( 'SimpleXMLElement' );
+	}
+
+	/**
+	 * @return bool
+	 */
+	public static function is_unfiltered_files_upload_enabled() {
+		return ! ! get_option( 'elementor_unfiltered_files_upload', false );
+	}
+
+	public static function is_upload_enabled() {
+		static $enabled = null;
+		if ( null === $enabled ) {
+			$enabled = self::is_unfiltered_files_upload_enabled() && self::file_sanitizer_can_run();
+			$enabled = apply_filters( 'elementor/files/svg/enabled', $enabled );
+		}
+		return $enabled;
+	}
+
+	public function support_unfiltered_files_upload( $existing_mimes ) {
+		$existing_mimes[ static::FILE_TYPE ] = static::MIME_TYPE;
+
+		return $existing_mimes;
+	}
+
+	/**
+	 * handle_upload_prefilter
+	 * @param $file
+	 *
+	 * @return mixed
+	 */
+	public function handle_upload_prefilter( $file ) {
+		if ( ! $this->is_elementor_media_upload() || static::MIME_TYPE !== $file['type'] ) {
```

Comment:
Where is the `static::MIME_TYPE`

Reply:
It's in the children handlers (svg-handler.php + json-handler.php).